### ISSUE TITLE
more odd.flag file out of generated folder 

### ISSUE THREAD

**Issue by KevinJump** [zpqrtbnk/Zbu.ModelsBuilder#131]
When you use AppData and ModelsDirectory options you can put your models into app_code, and you can then include them in your project and get all the goodies as you develop. 

However when you change a model ModelsBuilder creates a odd.flag file in the generated folder - to mark the models as out of date - as this folder is in app_code and part of your project it causes a site restart (a file change is detected) when it is written to the folder. 

could we have an option to move the odd.flag file to somewhere else? that way changing a doctype in the back office wouldn't cause the site to restart ?

**Comment by zpqrtbnk** [comment]
Makes sense. Some files should stay in `~/App_Data/Models` whatever `ModelsDirectory` is.
